dark/reads.py

```diff
@@ -190,13 +190,23 @@ class AARead(Read):
     """
     def properties(self):
         """
-        Translate an amino acid sequence to properties.
+        Translate an amino acid sequence to properties of the form:
+        'F': HYDROPHOBIC | AROMATIC.
 
         @return: A generator yielding properties for the residues in the
             current sequence.
         """
         return (PROPERTIES.get(aa, NONE) for aa in self.sequence)
 
+    def propertyDetails(self):
+        """
+        Translate an amino acid sequence to properties. Each property of the
+        amino acid gets a value scaled from -1 to 1.
+
+        @return: A list of property dictionaries.
+        """
+        return (PROPERTY_DETAILS.get(aa, NONE) for aa in self.sequence)
```

Comment:
Ah, shit, I thought it said `None`, not `NONE`. No, you'd better leave it in :-)   If you take it out just to see what happens, some tests should fail - if not our tests should be better!